examples/collect100.py

```diff
@@ -53,9 +53,9 @@ def collect(date, id_string, captures):
         print("loaded GDS")
         for x in range(1, capture_length + 1):
             print("entering capture", x)
-            ch1_waveform = g.waveform(ch=1)
+            ch2_waveform = g.waveform(ch=2)
             print("GOT WAVEFORM")
-            data = psd.concat([ch1_waveform], "ch").reorder("t")
+            data = psd.concat([ch2_waveform], "ch").reorder("t")
```

Comment:
I'm going to approve this PR b/c it's purpose is just to change the channel used for acquisition, but the only thing this line is doing is creating a singleton dimension called "ch".  This is a relic from earlier programs where we were acquiring more than one channel.

```
In [3]: a = nddata(r_[0:9],'t')                                                                                                        
                                                                                                                                       
In [4]: concat([a], "ch").reorder('t')
Out[4]: 
array([[0.+0.j],
       [1.+0.j],
       [2.+0.j],
       [3.+0.j],
       [4.+0.j],
       [5.+0.j],
       [6.+0.j],
       [7.+0.j],
       [8.+0.j]])
                +/-None
        dimlabels=['t', 'ch']
        axes={`t':array([0, 1, 2, 3, 4, 5, 6, 7, 8])
                        +/-None,
                `ch':array([0])
                        +/-None}

In [5]: concat([a], "ch").reorder('t').squeeze()
Out[5]: 
array([0.+0.j, 1.+0.j, 2.+0.j, 3.+0.j, 4.+0.j, 5.+0.j, 6.+0.j, 7.+0.j,
       8.+0.j])
                +/-None
        dimlabels=['t']
        axes={`t':array([0, 1, 2, 3, 4, 5, 6, 7, 8])
                        +/-None}
```